api/ServiceWorkerContainer.json

```diff
@@ -41,7 +41,7 @@
           "spec_url": "https://w3c.github.io/ServiceWorker/#navigator-service-worker-controller",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "54"
```

Comment:
I tracked this down to https://storage.googleapis.com/chromium-find-releases-static/77a.html#77a9444d1b574bb603e5a79d588e6f9e93361b1e

The `controller` property has been there since way before:
https://chromium.googlesource.com/chromium/src/+/d51c935c71766b703d7adca88ef1cdc947c90699

So this is giving an incorrect result because we're testing the prototype, but this property had `[Unforgeable]` at one point.

Reply:
Should this be left as "40" then?

Reply:
I've sent https://github.com/foolip/mdn-bcd-collector/pull/2283.

Reply:
https://mdn-bcd-collector.appspot.com/tests/api/ServiceWorkerContainer/controller is now updated, and I've confirmed 40 is the right version on BrowserStack.